test/IntegrationTests.py

```diff
@@ -20,18 +50,21 @@ def setUpClass(cls):
         super(IntegrationTests, cls).setUpClass()
 
         options = Options()
+        capabilities = DesiredCapabilities.CHROME
+        capabilities['loggingPrefs'] = {'browser': 'SEVERE'}
```

Comment:
I tried in vain to find docs for this, did you? If this config does what's needed here that's fine for this PR, I'm just curious if we might like other settings in other cases, to get messages that don't quite rise to the level of SEVERE :rage4:

Reply:
https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities

Available values for most loggers are "OFF", "SEVERE", "WARNING", "INFO", "CONFIG", "FINE", "FINER", "FINEST", "ALL".

Reply:
mmm yeah, I saw that one, but it also says

> See the documentation of each driver for what browser specific logging components are available.

Which I was hoping existed somewhere and would tell us something about what kind of messages are included in each level... oh well, trial and error I guess.